config/webpack.config.js

```diff
@@ -235,6 +237,10 @@ module.exports = env => {
           : `[name].[contenthash]-${timestamp}.css`,
       }),
       new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+      new CopyPlugin([
+        { from: 'src/site/assets/js/execute-axe-check.js', to: './' },
+        { from: 'src/site/assets/robots.txt', to: './' },
+      ]),
```

Comment:
Webpack will just move over the single js asset that we need for running the axe checker, and `robots.txt` because our build needs it.

Reply:
Might be worth adding a comment here explaining _why_ just these files are copied.

Also, I don't think you need the `to: './'`` since it's relative to the `output` directory anyhow, and [the default is `compiler.options.output`](https://webpack.js.org/plugins/copy-webpack-plugin/#to).